laser-py/laser.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# LASER  Language-Agnostic SEntence Representations
+# is a toolkit to calculate multilingual sentence embeddings
+# and to use them for document classification, bitext filtering
+# and mining
+#
+# --------------------------------------------------------
+#
+# Helper functions for tokenization and BPE
+
+import os
+import sys
+import logging
+import sentencepiece as spm
+from sacremoses import MosesPunctNormalizer, MosesDetokenizer
+
+logging.basicConfig(
+    stream=sys.stdout,
+    level=logging.INFO,
+    format="%(asctime)s | %(levelname)s | %(name)s | %(message)s")
+logger = logging.getLogger("preprocess")
+
+# get environment
+assert os.environ.get('LASER'), 'Please set the enviornment variable LASER'
+
+class LaserTokenizer:          
+    def tokenize(inp_fname, out_fname, spm_model_file, lang='en',
+                lower_case=True, descape=False,
+                verbose=False, over_write=False, gzip=False):
+        assert lower_case, 'lower case is needed by all the models'
```

Comment:
remove `lower_case` from the function arguments and also the assert above. We will be performing lowercasing regardless.

Reply:
I think we can remove the assert but keep `lower_case` as an argument (just in case someone reuses this tokenizer in a cased way, e.g. when distilling a new encoder). 

Also, I would move the arguments that configure the way test are processed (spm_model_file, lower_case, descape, and maybe also lang) to the `__init__` method, because we may want to initialize different tokenizers with different values of them, but we don't expect these parameters to change for a single given tokenizer.

Reply:
Okay yes let's keep `lower_case` then and move the args to the `__init__`.